pandas/core/series.py

```diff
@@ -251,6 +254,36 @@ class Series(base.IndexOpsMixin, generic.NDFrame):
     Note that the Index is first build with the keys from the dictionary.
     After this the Series is reindexed with the given Index values, hence we
     get all NaN as a result.
+    
+    Constructing Series from an array with `copy=False`.
```

Comment:
I'd call this a list, since the difference between an array and list in this context is important in terms of ability to share memory.